src/constants/AuthConstants.js

```diff
@@ -0,0 +1,15 @@
+const initialsignInData = {
+    email: "",
+    password: ""
+}
+const initialSignupData = {
+    firstName: "",
+    lastName: "",
+    email: "",
+    password: ""
+}
+const guestCredentials = {
+    email: "himmatsingh@gmail.com",
+    password: "specialops1.5"
+}
+export { initialsignInData, initialSignupData, guestCredentials };
```

Comment:
I really like how you are creating a separate file here for this data as including all this in login signup pages  would make it difficult to read and understand